suite-native/module-accounts/src/screens/AccountDetailScreen.tsx

```diff
@@ -16,23 +16,32 @@ import {
     selectAccountTransactions,
 } from '@suite-common/wallet-core';
 import { TransactionList } from '@suite-native/transactions';
+import {
+    selectEthereumAccountTokenTransactions,
+    selectEthereumAccountToken,
+} from '@suite-native/ethereum-tokens';
 
 import { TransactionListHeader } from '../components/TransactionListHeader';
 import { AccountDetailScreenHeader } from '../components/AccountDetailScreenHeader';
 
 export const AccountDetailScreen = memo(
     ({ route }: StackProps<AccountsStackParamList, AccountsStackRoutes.AccountDetail>) => {
-        const { accountKey } = route.params;
+        const dispatch = useDispatch();
+        const { accountKey, tokenSymbol } = route.params;
         const account = useSelector((state: AccountsRootState) =>
             selectAccountByKey(state, accountKey),
         );
         const accountLabel = useSelector((state: AccountsRootState) =>
             selectAccountLabel(state, accountKey),
         );
         const accountTransactions = useSelector((state: TransactionsRootState) =>
-            selectAccountTransactions(state, accountKey),
+            tokenSymbol
+                ? selectEthereumAccountTokenTransactions(state, accountKey, tokenSymbol)
+                : selectAccountTransactions(state, accountKey),
```

Comment:
are you sure this will not break memoization? I'm not sure but maybe it's okay

Reply:
shoudln't be issue for memoization but I would prefer to have this wrapped in new selector anyway

Reply:
new selector created in https://github.com/trezor/trezor-suite/commit/4d5c05278740cdd3d9e85981c1257cf8feac0a25